frontend/src/data/variables/AcsPopulationProvider.ts

```diff
@@ -54,7 +54,10 @@ class AcsPopulationProvider extends VariableProvider {
     throw new Error("Not implemented");
   }
 
-  async getDataInternal(breakdowns: Breakdowns): Promise<MetricQueryResponse> {
+  async getDataInternal(
+    metricQuery: MetricQuery
+  ): Promise<MetricQueryResponse> {
+    const breakdowns = metricQuery.breakdowns;
```

Comment:
I forgot, where did we land on the "additional columns" aspect? Are we okay with returning additional columns that were not requested in the query or should we strip them out? Maybe we should add a method spec for getData() that says how it behaves

Reply:
I'm inclined to remove them. I think I'll add a TODO and follow up :)